app/assets/stylesheets/components/_madlib-leadin.scss

```diff
@@ -0,0 +1,9 @@
+.madlib-leadin {
+  color: $medium-gray;
```

Comment:
just checking to see if it makes sense to set a style/variable/mixin for texts of this hierarchal level